packages/oscal-react-library/src/components/OSCALControl.tsx

```diff
@@ -241,9 +241,139 @@ const OSCALControl: React.FC<OSCALControlProps> = (props) => {
     );
   }
 
+  const position = props?.modificationAlters
+    ?.find((item) => item["control-id"] === props.control.id)
+    ?.adds?.flatMap((item) => item["position"])
+    .filter((part) => part)[0];
+
   const label = propWithName(control.props, "label")?.value;
   const controlOrParentWithdrawn = withdrawn || isWithdrawn(control);
 
+  if (position === "after") {
```

Comment:
Can you add `import { Position } from "@easydynamics/oscal-types";` to the top of this file and then use the enum variants in these conditions (`if (position === Position.AFTER) {`)?

I do think we need to find a slightly less-repetitive way to handle the code within the conditions but I am not quite sure how to go about that refactoring. I think @tuckerzp is pretty familiar with this so maybe he can provide an opinion

Reply:
Thanks @kylelaker for this suggestion, I will do that.

Reply:
> a slightly less-repetitive way to handle the code within the conditions

Briefly looking at this, the only difference I see here for each if is the position of `modifcationDisplay` in the component being created.

At the very least, could we not do something like:

```TypeScript
<OSCALControlCard>
  <position1>
    {position === Position.AFTER) ? modificationDisplay : null}
  </position1>
  <position2>
    {position === Position.Before) ? modificationDisplay : null}
  </position2>
</OSCALControlCard>
```

I don't think we need to create a completely different declaration of the component if we are just switching 1 thing around based on an if statement.

Reply:
Thanks @tuckerzp, I refactored the code. 
This is the profile I am using to test the implementation. https://raw.githubusercontent.com/EasyDynamics/oscal-demo-content/test-content/profiles/NIST_SP-800_rev4TestProfile.json
These are few screenshots
<img width="1009" alt="image" src="https://github.com/EasyDynamics/oscal-react-library/assets/59936247/8b753e8b-26ef-400b-ad80-654086d5ccb3">

<img width="1022" alt="image" src="https://github.com/EasyDynamics/oscal-react-library/assets/59936247/eeb551c1-783c-42d1-9da7-5d32acbe085d">